data/guake-autoconfigure.desktop.in

```diff
@@ -1,6 +1,6 @@
 [Desktop Entry]
-Name=Autoconfigure Guake
-Comment=Autoconfigure the start of the Guake terminal
+_Name=Autoconfigure Guake
+_Comment=Autoconfigure the start of the Guake terminal
```

Comment:
_ makes gettext pick up these strings for translation. I'm not sure if there is any additional automake magic to make it work, but that's the basics.